Task2/aga/TicTacToe/TicTacToe/TheGame.xaml.cs

```diff
@@ -0,0 +1,242 @@
+
+        }
+
+        private void A2_Click(object sender, RoutedEventArgs e)
+        {
+            Button b = (Button)sender;
+            int turn = click.GetTurn();
+            int param = 1;
+
+            displayButton(turn, b, param);
+        }
+
+        private void A3_Click(object sender, RoutedEventArgs e)
+        {
+            Button b = (Button)sender;
+            int turn = click.GetTurn();
+            int param = 2;
+
+            displayButton(turn, b, param);
+        }
+
+        private void B1_Click(object sender, RoutedEventArgs e)
+        {
+            Button b = (Button)sender;
+            int turn = click.GetTurn();
+            int param = 3;
+
+            displayButton(turn, b, param);
+        }
+
+        private void B2_Click(object sender, RoutedEventArgs e)
+        {
+            Button b = (Button)sender;
+            int turn = click.GetTurn();
+            int param = 4;
+
+            displayButton(turn, b, param);
+        }
+
+        private void B3_Click(object sender, RoutedEventArgs e)
+        {
+            Button b = (Button)sender;
+            int turn = click.GetTurn();
+            int param = 5;
+
+            displayButton(turn, b, param);
+        }
+
+        private void C1_Click(object sender, RoutedEventArgs e)
+        {
+            Button b = (Button)sender;
+            int turn = click.GetTurn();
+            int param = 6;
+
+            displayButton(turn, b, param);
+        }
+
+        private void C2_Click(object sender, RoutedEventArgs e)
+        {
+            Button b = (Button)sender;
+            int turn = click.GetTurn();
+            int param = 7;
+
+            displayButton(turn, b, param);
+        }
+        private void C3_Click(object sender, RoutedEventArgs e)
+        {
+            Button b = (Button)sender;
+            int turn = click.GetTurn();
+            int param = 8;
+
+            displayButton(turn, b, param);
+        }
+
+        private void displayTurn(int t)
+        {
+            Label l = diplayTurn;
+            l.Content = t;
+        }
+
+        private void displayMessageBox(int param, int turn)
+        {
+            Label l = messageBox;
+            
+            if (wasWinned())
+                { l.Content = "The winner is "+ winner; }
+            else if (turn == 9)
+                {
+                    l.Content = "It's a draw!";
+                    tryAgain.Visibility = Visibility.Visible;
+                }
+            else if (wasClicked(param))
+            { l.Content = "Choose different field"; }
+            else
+                {
+                    if (turn % 2 != 0)
+                    {
+                        l.Content = "Player O turn";
+                    }
+                    else
+                    {
+                        l.Content = "Player X turn";
+                    }
+                }
+        }
+
+        private void displayButton(int turn, Button b, int param)
```

Comment:
Does displayButton method only display button or does it do something more? Naming things right is one of the most important things.